tools/taskcluster/get-codecov-token.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script talks to the taskcluster secrets service to obtain the
```

Comment:
FYI: it looks like a-c made a generic version of this script – https://searchfox.org/mozilla-mobile/source/android-components/automation/taskcluster/helper/get-secret.py#23 – which is shared by Fenix but we're still using our own split up versions (like this script). Ideally, we'd be sharing this code (e.g. in [the shared gradle plugin](https://github.com/mozilla-mobile/android-automation-tools/tree/master/gradle-plugin)) but there's a lot of friction to doing so so we haven't been.

Sharing this build/infra code is a non-trivial problem that I don't know how to solve and I want to make sure you're aware of it too!